safe-svg.php

```diff
@@ -500,12 +500,24 @@ protected function svg_dimensions( $svg ) {
                  *
                  * @return {bool} If we should use the width & height attributes first or not.
                  */
-                if ( (bool) apply_filters( 'safe_svg_use_width_height_attributes', false, $svg ) ) {
-                    $width  = $attr_width;
-                    $height = $attr_height;
+                $use_width_height = (bool) apply_filters( 'safe_svg_use_width_height_attributes', false, $svg );
+
+                if ( $use_width_height ) {
+                    if ( isset( $attr_width, $attr_height ) ) {
+                        $width  = $attr_width;
+                        $height = $attr_height;
+                    } elseif ( isset( $viewbox_width, $viewbox_height ) ) {
+                        $width  = $viewbox_width;
+                        $height = $viewbox_height;
+                    }
                 } else {
-                    $width  = $viewbox_width;
-                    $height = $viewbox_height;
+                    if ( isset( $viewbox_width, $viewbox_height ) ) {
+                        $width  = $viewbox_width;
+                        $height = $viewbox_height;
+                    } elseif ( isset( $attr_width, $attr_height ) ) {
+                        $width  = $attr_width;
+                        $height = $attr_height;
+                    }
                 }
 
                 if ( ! $width && ! $height ) {
```

Comment:
```suggestion
                if ( ! $width || ! $height ) {
```

Reply:
The current condition requires both `$width` and `$height` to be non-zero (as they initially set at [:471](https://github.com/10up/safe-svg/pull/51/files#diff-05bf19b873cfb6358cd7c239366db6675635bf4b03a4b56353102e359d5c2859L471)). The suggested condition will drop this to "any is zero" which will return false for SVG with zero size of one side.

Even if this is not related to the current PR, probably needs improvements:
- `$width` and `$height` initial assign to `false` at [:471-472](https://github.com/10up/safe-svg/pull/51/files#diff-05bf19b873cfb6358cd7c239366db6675635bf4b03a4b56353102e359d5c2859L471)
- Checking any or both are `false` at [:511](https://github.com/10up/safe-svg/pull/51/files#diff-05bf19b873cfb6358cd7c239366db6675635bf4b03a4b56353102e359d5c2859L511)

Reply:
@jerturowetz The current condition expects both `$width` and `$height` to be non-zero (as they initially set at :471).

The suggested change will drop this to "any is zero" condition which will return cause function return false in case of one side SVG is zero.